index.js

```diff
@@ -36,6 +36,41 @@ const AD_TYPES_VERSION = 1; // Used with the ad type slugs
 const ATTR_PREFIX = "data-ea-";
 const ABP_DETECTION_PX = "https://media.ethicalads.io/abp/px.gif";
 
+// Verbosity and logging
+//
+// Set with:
+//
+//  <div data-ea-publisher="..." data-ea-verbosity="quiet"></div>
+const VERBOSITY = {
+  quiet: 0, // Errors only
+  normal: 1, // Warnings only (default)
+  verbose: 2, // Debug messages
+};
+const logger = {
+  verbosity: VERBOSITY["normal"], // Default
+
+  debug(message, ...params) {
+    if (this.verbosity >= 2) {
```

Comment:
These should use the VERBOSITY names, and not ints, right? Seems more explicit.

Reply:
```suggestion
    if (this.verbosity >= VERBOSITY["verbose"]) {
```

Reply:
Done